tomviz/BadStackDialog.cxx

```diff
@@ -0,0 +1,39 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#include "BadStackDialog.h"
+#include "ui_BadStackDialog.h"
+
+#include "ImageStackModel.h"
+
+#include <QDialog>
+
+namespace tomviz {
+
+BadStackDialog::BadStackDialog(QWidget* parent, ImageStackModel* tableModel)
```

Comment:
We should probably rename this TIFFImageStackDialog or something a little more generic as we add more to this.

Reply:
Fixed in https://github.com/OpenChemistry/tomviz/pull/1544/commits/b6b6de111a753b94f0043cfd889becfbd6d0e257